pkg/cmd/repo/garden/garden.go

```diff
@@ -0,0 +1,412 @@
+	out := opts.IO.Out
+
+	if runtime.GOOS == "windows" {
+		return errors.New("sorry :( this command only works on linux and macos")
+	}
+
+	if !opts.IO.IsStdoutTTY() {
+		return errors.New("must be connected to a terminal")
+	}
+
+	httpClient, err := opts.HttpClient()
+	if err != nil {
+		return err
+	}
+
+	var toView ghrepo.Interface
+	apiClient := api.NewClientFromHTTP(httpClient)
+	if opts.RepoArg == "" {
+		var err error
+		toView, err = opts.BaseRepo()
+		if err != nil {
+			return err
+		}
+	} else {
+		var err error
+		viewURL := opts.RepoArg
+		if !strings.Contains(viewURL, "/") {
+			currentUser, err := api.CurrentLoginName(apiClient, ghinstance.Default())
+			if err != nil {
+				return err
+			}
+			viewURL = currentUser + "/" + viewURL
+		}
+		toView, err = ghrepo.FromFullName(viewURL)
+		if err != nil {
+			return fmt.Errorf("argument error: %w", err)
+		}
+	}
+
+	seed := computeSeed(ghrepo.FullName(toView))
+	rand.Seed(seed)
+
+	termWidth, termHeight, err := utils.TerminalSize(out)
+	if err != nil {
+		return err
+	}
+
+	termWidth -= 10
+	termHeight -= 10
+
+	geo := &Geometry{
+		Width:      termWidth,
+		Height:     termHeight,
+		Repository: toView,
+		// TODO based on number of commits/cells instead of just hardcoding
+		Density: 0.3,
+	}
+
+	maxCommits := geo.Width * geo.Height
+
+	commits, err := getCommits(httpClient, toView, maxCommits)
+	if err != nil {
+		return err
+	}
+	player := &Player{0, 0, utils.Bold("@"), geo, 0}
+
+	garden := plantGarden(commits, geo)
+	clear(opts.IO)
+	drawGarden(out, garden, player)
+
+	// thanks stackoverflow https://stackoverflow.com/a/17278776
+	if runtime.GOOS == "darwin" {
+		_ = exec.Command("stty", "-f", "/dev/tty", "cbreak", "min", "1").Run()
+		_ = exec.Command("stty", "-f", "/dev/tty", "-echo").Run()
+	} else {
+		_ = exec.Command("stty", "-F", "/dev/tty", "cbreak", "min", "1").Run()
+		_ = exec.Command("stty", "-F", "/dev/tty", "-echo").Run()
+	}
+
+	var b []byte = make([]byte, 1)
+	for {
+		_, err := opts.IO.In.Read(b)
+		if err != nil {
+			return err
+		}
+
+		quitting := false
+		switch {
+		case isLeft(b):
+			player.move(DirLeft)
+		case isRight(b):
+			player.move(DirRight)
+		case isUp(b):
+			player.move(DirUp)
+		case isDown(b):
+			player.move(DirDown)
+		case isQuit(b):
+			quitting = true
+		}
+
+		if quitting {
+			break
+		}
+
+		clear(opts.IO)
+		drawGarden(out, garden, player)
+	}
+
+	fmt.Println()
+	fmt.Println(utils.Bold("You turn and walk away from the wildflower garden..."))
+
+	return nil
+}
+
+// TODO fix arrow keys
+
+func isLeft(b []byte) bool {
+	return bytes.EqualFold(b, []byte("a")) || bytes.EqualFold(b, []byte("h"))
+}
+
+func isRight(b []byte) bool {
+	return bytes.EqualFold(b, []byte("d")) || bytes.EqualFold(b, []byte("l"))
+}
+
+func isDown(b []byte) bool {
+	return bytes.EqualFold(b, []byte("s")) || bytes.EqualFold(b, []byte("j"))
+}
+
+func isUp(b []byte) bool {
+	return bytes.EqualFold(b, []byte("w")) || bytes.EqualFold(b, []byte("k"))
+}
```

Comment:
This small change would support <kbd>Z</kbd> <kbd>Q</kbd> <kbd>S</kbd> <kbd>D</kbd> for French/AZERTY keyboards...

```suggestion
func isLeft(b []byte) bool {
	return bytes.EqualFold(b, []byte("a")) || bytes.EqualFold(b, []byte("q")) || bytes.EqualFold(b, []byte("h"))
}

func isRight(b []byte) bool {
	return bytes.EqualFold(b, []byte("d")) || bytes.EqualFold(b, []byte("l"))
}

func isDown(b []byte) bool {
	return bytes.EqualFold(b, []byte("s")) || bytes.EqualFold(b, []byte("j"))
}

func isUp(b []byte) bool {
	return bytes.EqualFold(b, []byte("w")) || bytes.EqualFold(b, []byte("z")) || bytes.EqualFold(b, []byte("k"))
}
```

**For reference**

[Google results for`azerty zqsd controls OR games`](https://www.google.com/search?bih=835&biw=1821&hl=en&ei=HLpWX_2qJInx-gSw1b-IAw&q=azerty+zqsd+controls+OR+games&oq=azerty+zqsd+controls+OR+games&gs_lcp=CgZwc3ktYWIQAzIFCCEQoAE6BwghEAoQoAE6BQgAEM0COgUIIRCrAlCoG1iWLWCGLmgAcAB4AIAB8gKIAbsYkgEIMS4xMi4zLjKYAQCgAQGqAQdnd3Mtd2l6wAEB&sclient=psy-ab&ved=0ahUKEwj9qNvKkdjrAhWJuJ4KHbDqDzEQ4dUDCA0&uact=5) 🔍 

And example ⌨️ 

![image](https://user-images.githubusercontent.com/121322/92419599-5f1b5e80-f123-11ea-888e-331fc13ca241.png)